docs/flows/flows.puml

```diff
@@ -70,33 +72,52 @@ autonumber "Bond:[00]"
 
 alice -> forum: create forum proposal
 alice <- guarantor: positive/negative feedback
-alice -> bondcontract: if (positive) create bond
+alice -> bondcontract: if (positive) create bond (specify reward mechanism)
 
-guarantor -> bondcontract: contribute
+guarantor -> bondcontract: contribute BIT tokens
+bondcontract <-- guarantor: ERC20 Fungible bond tokens (1:1 swap)
```

Comment:
Are we specifying here whether the fungible token is fungible within the bond contract or fungible within all bond contracts?
_(For secondary markets, I can see the value in keeping each bond contract as unique pools if fungible tokens)_

Reply:
We can scope token fungibility to within the bond contract; a swap can be used to be fungible within another bond contract. This view can be translated into fungibility within all bond contracts by swapping tokens for another derivative which is an index across all bond contracts.

Reply:
Sounds good 👍